src/editor/main.js

```diff
@@ -170,7 +169,12 @@ button('delete', () => {
   radio.closest('.layer').remove();
 });
 button('export', async () => {
-  exportSizeArray.forEach(async (size) => {
+  
+  const exportSizes = Array.from(document.forms['exportSizes'].elements['sizes'])
```

Comment:
```suggestion
  const exportSizes = new FormData(document.forms['exportSizes']).getAll('sizes')
```
You can remove the `filter` line like this